tests/virtualization/virtman_view.pm

```diff
@@ -50,8 +50,13 @@ sub run {
     assert_screen 'virt-manager';
 
     # go to view now
-    wait_screen_change { send_key 'alt-v' };
-    wait_screen_change { send_key 'right' };
+    # decompose alt-v - poo#54056
+    hold_key 'alt';
+    send_key 'v';
+    sleep(1);
+    release_key 'alt';
+    send_key 'right';
+    assert_screen 'virtman-viewmenu-graph';
```

Comment:
From the code I can see that this module is used for SLES too, so we might need to create needle for it too. Let me check.

Reply:
I've triggered https://openqa.suse.de/tests/3568360 (paste it here not to loose the link), will use it to create the needle as you did for TW and then we can merge this PR.